### PR TITLE
FAT-4733

### PR DESCRIPTION
**Pull request by nayimovag**
autotest updates according to updates in test case [C350928](https://foliotest.testrail.io/index.php?/cases/view/350928)

https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1220

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/dictionary/permissions.js [hunk 1]
```diff
@@ -92,7 +92,6 @@ export default {
   uiUsersDeclareItemLost: { internal: 'ui-users.loans.declare-item-lost', gui: 'Users: User loans declare lost' },
   usersLoansRenewThroughOverride: { internal: 'ui-users.loans.renew-override', gui: 'Users: User loans renew through override' },
   uiUsersEditProfile: { internal: 'ui-users.edit', gui: 'Users: Can edit user profile' },
-  uiUsersViewProfile: { internal: 'ui-users.view', gui: 'Users: Can view user profile' },
```
Comment (penfriend) [inline cypress/support/dictionary/permissions.js:95]: Why do you remove this permission? Other test reuse this permission, so another test will fail after this removal of the permission

**Review by penfriend — CHANGES_REQUESTED** [review]

**Comment by penfriend** [comment]
please add the link to jenkins run with autotest run

File: cypress/support/dictionary/permissions.js (hunk 1 above)
Comment (penfriend) [reply cypress/support/dictionary/permissions.js:95]: Please update this permission in all autotests where this permission is used.

Comment (nayimovag) [reply cypress/support/dictionary/permissions.js:95]: this permission is a duplicate of "uiUsersView: { internal: 'ui-users.view', gui: 'Users: Can view user profile' }". There was only one occurence in "cypress/e2e/bulk-edit/permissions/bulk-edit-user-permissions.cy.js" and it was replaced as you can see

**Review by penfriend — APPROVED** [review]

**Pull request merged**